dxr/plugins/clang/dxr-index.cpp

```diff
@@ -1087,7 +1095,7 @@ class IndexConsumer : public ASTConsumer,
             text[i] != '\t' && text[i] != '\n')
           text[i] = '?';
       }
-      recordValue("text", text, true);
+      macromap.insert(make_pair(nameStart, text));
```

Comment:
Nope! Sold. I didn't know you had compiled m-c under it, nor did I notice that it was in an object that's discarded per translation unit. Your back-of-the-envelope calculations look good as well; that's exactly what I wanted to see. :-) It'll be a tad faster in C++ anyway, although we'll pay a little more IO since we're repetitively writing the macro definition out to the CSVs on every reference. We'll go with this! Thanks!